.ci/docker/centos-rocm/Dockerfile

```diff
@@ -115,13 +115,6 @@ COPY triton_version.txt triton_version.txt
 RUN if [ -n "${TRITON}" ]; then bash ./install_triton.sh; fi
 RUN rm install_triton.sh common_utils.sh triton.txt triton_version.txt
 
-# Install AOTriton (Early fail)
-COPY ./aotriton_version.txt aotriton_version.txt
-COPY ./common/common_utils.sh common_utils.sh
-COPY ./common/install_aotriton.sh install_aotriton.sh
```

Comment:
Need to remove `.ci/docker/aotriton_version.txt` and `.ci/docker/common/install_aotriton.sh` as well